osu.Game.Rulesets.Osu/Mods/OsuModAlternate.cs

```diff
@@ -89,9 +102,22 @@ private bool checkCorrectAction(OsuAction action)
             {
                 // User alternated correctly.
                 lastActionPressed = action;
+                numActions = 0;
                 return true;
             }
 
+            if (lastActionPressed == action)
+            {
+                // If same action, increment counter.
```

Comment:
This comment doesn't bring any value, it just restates the code. Comments should not be talking about what is being done, they should be talking about *why* it is being done.